docs/application/tizen-studio/TOC.md

```diff
@@ -1,149 +1,149 @@
-### [Web Simulator Features](web-tools/web-simulator-features.md)
-## [CLI](common-tools/command-line-interface.md)
-## [Keyboard Shortcuts](common-tools/keyboard-shortcuts.md)
+#### [Overview](/application/tizen-studio/common-tools/emulator.md)
+#### [Emulator Manager](/application/tizen-studio/common-tools/emulator-manager.md)
+#### [Emulator Control Panel](/application/tizen-studio/common-tools/emulator-control-panel.md)
+#### [Extended Emulator Features](/application/tizen-studio/common-tools/emulator-features.md)
+### [Web Unit Test Tool](/application/tizen-studio/web-tools/web-unit-test-tool.md)
+### [Us the Web Simulator](/application/tizen-studio/web-tools/web-simulator.md)
+### [Web Simulator Features](/application/tizen-studio/web-tools/web-simulator-features.md)
+## [CLI](/application/tizen-studio/common-tools/command-line-interface.md)
+## [Keyboard Shortcuts](/application/tizen-studio/common-tools/keyboard-shortcuts.md)
 
 # Platform Tools
-## [Overview](platform-tools/overview.md)
-## [Platform Development Tools](platform-tools/platform-tools.md)
+## [Overview](/application/tizen-studio/platform-tools/overview.md)
+## [Platform Development Tools](/application/tizen-studio/platform-tools/platform-tools.md)
 
-# [Configurable SDK](configurable-sdk/configurable-sdk.md)
+# [Configurable SDK](/application/tizen-studio/configurable-sdk/configurable-sdk.md)
 
 # Extension SDK
-## [Overview](extension-sdk/overview.md)
-## [In-App Purchase Programing Guide](extension-sdk/in-app-purchase.md)
-## [InMobi Tizen Ad SDK Programing Guide](extension-sdk/inmovi.md)
-## [VMAX Tizen Ad SDK Integration Guide](extension-sdk/vmax.md)
+## [Overview](/application/tizen-studio/extension-sdk/overview.md)
+## [In-App Purchase Program Guide](/application/tizen-studio/extension-sdk/in-app-purchase.md)
+## [InMobi Tizen Ad SDK Program Guide](/application/tizen-studio/extension-sdk/inmovi.md)
+## [VMAX Tizen Ad SDK Integration Guide](/application/tizen-studio/extension-sdk/vmax.md)
 
 # RT IDE
-## [Overview](rt-ide/overview.md)
-## Getting Started
-### [Overview](rt-ide/getting-started/overview.md)
-### [Installing Tizen Studio for RT](rt-ide/getting-started/install.md)
-### [Managing a Project](rt-ide/getting-started/create.md)
-### [Flashing the Project](rt-ide/getting-started/flash.md)
-### [Using the Serial Terminal](rt-ide/getting-started/terminal.md)
-### [Debugging the Project](rt-ide/getting-started/debug.md)
-## SmartThings&trade; Extension
-### [Overview](rt-ide/smartthings-ext/overview.md)
-### [Creating a SmartThings&trade; Project](rt-ide/smartthings-ext/create-smartthings.md)
-### [Building the SmartThings&trade; Project](rt-ide/smartthings-ext/build-smartthings.md)
-### [Managing SmartThings&trade; Certificates](rt-ide/smartthings-ext/certificate-smartthings.md)
-### [Device/Resource Model Manager](rt-ide/smartthings-ext/model-manager.md)
+## [Overview](/application/tizen-studio/rt-ide/overview.md)
+## Gett Started
```

Comment:
It looks like `Get` is correct.

Reply:
Getting started is correct. We will change a gerund to a verb next time.